### DISCUSSION TITLE
RSS feed for bun.sh/blog ?

### DISCUSSION THREAD

**Discussion by andrewrothman** [oven-sh/bun#1663] [Q&A] [answered]
I'd love to stay up to date with Bun developments via my RSS feed reader. If I can do anything to help please let me know.

Thanks!
Andrew

**Comment by Jarred-Sumner** [comment]
This is a good idea

**Comment by knightspore** [comment]
I can't find the website for contributing (not sure if I'm missing something) but I'd be happy to make a PR for. this feature.

**Comment by JeremyEffinger** [comment]
Bump on this, trying to setup my news feed today and no RSS for Bun.

**Comment by Kayno0o** [comment]
I think using GitHub releases is a good way to achieve what you're looking for:
[oven-sh/bun/releases.atom](https://github.com/oven-sh/bun/releases.atom)

**Reply by RobertBorg** [reply, marked as answer]
for other ppl ending up here via searching. there is a feed at https://bun.sh/rss.xml
